README.md

```diff
@@ -107,23 +107,42 @@ You can configure caluma_interval with environment variables or CLI arguments, w
 Cli arguments take precedence.
 
 ```
-usage: __main__.py [-h] [-c STRING] [-d] [-v]
+usage: __main__.py [-h] [-c STRING] [-i STRING] [-s STRING] [-u STRING] [-d]
+                   [-v]
 
 Caluma companion app for handling intervalled forms
 
 optional arguments:
   -h, --help            show this help message and exit
   -c STRING, --caluma-uri STRING
                         defaults to "http://caluma:8000/graphql"
+  -i STRING, --oidc-client-id STRING
+  -s STRING, --oidc-client-secret STRING
+  -u STRING, --oidc-token-uri STRING
   -d, --debug           print debug messages
   -v, --version         show program's version number and exit
 ```
 
 The corresponding environment varibles are:
  * CALUMA_URI
+ * OIDC_CLIENT_ID
+ * OIDC_CLIENT_SECRET
+ * OIDC_TOKEN_URI
 
+To enable oidc authentication over plain http (only for testing!), set
 
-## TODO: authorization
+```shell
+OAUTHLIB_INSECURE_TRANSPORT = 1
```

Comment:
In shell, you don't use spaces for variable assignments (for people who copy&paste into their shell :P) Also this probably needs an `export` too, otherwise the python process won't see the variable

Reply:
It was a mistake to put this in a shell-code-block. We're actually setting this in the docker-compose.override.yml.

It's fixed.